src/stages/stage5/timeline.c

```diff
@@ -400,17 +366,114 @@ static int stage5_superbullet(Enemy *e, int t) {
 	return 1;
 }
 
-void stage5_events(void) {
-	TIMER(&global.timer);
+TASK(greeter_fairy_attack, {
+		BoxedEnemy e;
+		int count;
+}) {
+	Enemy *e = TASK_BIND(ARGS.e);
+	cmplx dir = cnormalize(global.plr.pos - e->pos);
+
+	for(int i = -ARGS.count; i <= ARGS.count; i++) {
+		PROJECTILE(
+			.proto = pp_bullet,
+			.pos = e->pos,
+			.color = RGB(0.0, 0.0, 1.0),
+			.move = move_asymptotic_simple(3.5 * cdir(carg(dir) + 0.06 * i), 5),
+		);
+		play_sfx("shot1");
+	}
+}
+
+TASK(greeter_fairy, {
+		cmplx pos;
+		MoveParams move_enter;
+		MoveParams move_exit;
+}) {
+	Enemy *e = TASK_BIND(espawn_fairy_blue_box(ARGS.pos, ITEMS(.points = 1)));
+	e->move = ARGS.move_enter;
+
+	for (int i = 0; i < 5; i++) {
+		INVOKE_SUBTASK_DELAYED(80, greeter_fairy_attack, {
+			.e = ENT_BOX(e),
+			.count = difficulty_value(1, 2, 3, 4),
+		});
+		WAIT(20);
+	}
+
+	WAIT(200);
+
+	e->move = ARGS.move_exit;
+
+	STALL;
```

Comment:
The `STALL` is not needed in this case. It's generally used to keep subtasks alive, but:
* All subtasks invoked here are short enough that they'll surely be finished before this code is reached;
* If you implement my earlier suggestion, there wouldn't be subtasks at all.